analyzers/its/expected/Net5/Net5--net5.0-S3881.json

```diff
@@ -40,18 +40,18 @@
 {
 "uri":  "sources\Net5\Net5\S2952.Part1.cs",
```

Comment:
was there any change in this IT? I see that some lines moved (line 6 was at the end of the file)

Reply:
Although there were no changes in the ITs, we had an existing case with partial methods for which the behavior changed. 

You can find it here: https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Net5/Net5/S2952.Part1.cs

Raised by the old version:
Issue 1: Primary (line 6) + Secondary (line 6)
Issue 2: Primary (line 19) + Secondary (line 19)
Issue 3: Primary (line 28) + Secondary (line 28)

Raised by the new version:
Issue 1: Primary (line 6) + Secondary (line 6)
Issue 2: Primary (line 19) + Secondary (line 19) + Secondary (line 24)
Issue 3: Primary (line 28) + Secondary (line 28) + Secondary (line 33)

The difference is the additional secondary location with the message:
> 'ResourceHolder2.Dispose()' should call 'Dispose(true)' and 'GC.SuppressFinalize(this)'.

This was previously raised for "normal" methods but not for partial ones.